source.go

```diff
@@ -171,23 +189,32 @@ func (s *Source) nextPage(ctx context.Context) (sdk.Record, error) {
 }
 
 // getChildren gets all the child and grand-child blocks of the input block
-func (s *Source) getChildren(ctx context.Context, blockID string) ([]notion.Block, error) {
+func (s *Source) getChildren(ctx context.Context, block notion.Block) ([]notion.Block, error) {
+	if block.GetType() == notion.BlockTypeUnsupported {
+		// skip children of unsupported block types
+		sdk.Logger(ctx).Warn().
+			Str("block_type", block.GetType().String()).
+			Str("block_id", block.GetID().String()).
+			Msg("skipping children of unsupported block")
+		return []notion.Block{}, nil
+	}
```

Comment:
IIUC, we want to skip children of a block that is of `unsupported` type, but we still want the block itself?

Reply:
@hariso yup! In this case, I used the default `Brainstorm` template from Notion. The `unsupported` block type came from the block to add a new template in the screenshot below in `Add a New Topic`. The plaintext from that block may still be relevant, where the child of the block should not be iterated on.
![Screenshot 2023-01-04 at 8 28 59 AM](https://user-images.githubusercontent.com/6003000/210602902-596b471d-f9bb-468b-88c0-5749065c0e6a.png)